core/src/test/scala/sttp/tapir/generic/SchemaGenericAutoTest.scala

```diff
@@ -244,7 +244,15 @@ class SchemaGenericAutoTest extends AsyncFlatSpec with Matchers {
       )
     )
 
-    schemaType.asInstanceOf[SCoproduct[Entity]].discriminator shouldBe Some(SDiscriminator(FieldName("who_am_i"), Map.empty))
```

Comment:
would be great to add a test with non-identity subtype name transformations

Reply:
Will do!